charts/datadog/templates/_container-process-agent.yaml

```diff
@@ -39,6 +39,10 @@
       value: {{ .Values.agents.containers.processAgent.logLevel | default .Values.datadog.logLevel | quote }}
     - name: DD_SYSTEM_PROBE_ENABLED
       value: {{ .Values.datadog.networkMonitoring.enabled | quote }}
+    {{- if eq (include "should-enable-system-probe" .) "true" }}
+    - name: DD_SYSTEM_PROBE_EXTERNAL
```

Comment:
The existence of this env var is for a specific scenario where:
1. The default agent container and *entrypoint* is used, where multiple processes are run in the same container
2. The system-probe is run as a separate container, since it requires higher privileges. This env var instructs the "main" container to not run system-probe (more specifically, system-probe to exit on startup), since it will exist outside of that container.

In the helm chart, the `command` is being specified for each container AND each process has its own container. Thus the default entrypoint is not being used, and there is no danger of a `system-probe` running in the `process-agent` container.

In summary, I don't think this change is necessary for the helm chart. If you think our public documentation is insufficient in some regard, please let us know and we can work on getting that updated/clarified.